api/src/modules/widgets/page-filters.service.ts

```diff
@@ -0,0 +1,18 @@
+import * as fs from 'fs';
+import { Injectable } from '@nestjs/common';
+import { PageFilter } from '@shared/dto/widgets/page-filter.dto';
+
+@Injectable()
+export class PageFiltersService {
+  private filters: PageFilter[];
+
+  public constructor() {
+    this.filters = JSON.parse(
+      fs.readFileSync('data/filters/filters.json', 'utf-8'),
+    );
+  }
+
+  public async search(): Promise<PageFilter[]> {
```

Comment:
```suggestion
  public async getFilters(): Promise<PageFilter[]> {
```
Maybe would be a more comprehensive naming as we are not performing a lookup over the filters.
Then we could have another method to update the filters dynamically based on selections made in the FE

Reply:
I think this is a matter of choice, I can change this if you prefer getFilters.

Reply:
It's just s suggestion, I don't have strong opinions. Go ahead if you feel seach is better :)